ext/pdo_mysql/tests/pdomysql_002.phpt

```diff
@@ -0,0 +1,54 @@
+--TEST--
+PDO_mysql connect through PDO::connect
+--EXTENSIONS--
+pdo_mysql
+--SKIPIF--
+<?php
+require_once __DIR__ . '/inc/mysql_pdo_test.inc';
+MySQLPDOTest::skip();
+?>
+--FILE--
+<?php
+
+require_once __DIR__ . '/inc/mysql_pdo_test.inc';
+
+$db = Pdo::connect(PDO_MYSQL_TEST_DSN, PDO_MYSQL_TEST_USER, PDO_MYSQL_TEST_PASS);
+if (!$db instanceof PdoMysql) {
+    echo "Wrong class type. Should be PdoMysql but is " . get_class($db) . "\n";
+}
+
+$db->exec('CREATE TABLE IF NOT EXISTS pdomysql_002(id int NOT NULL PRIMARY KEY, name VARCHAR(10))');
```

Comment:
I don't think it needs to confirm the existence.

```suggestion
$db->exec('CREATE TABLE pdomysql_002(id int NOT NULL PRIMARY KEY, name VARCHAR(10))');
```

Reply:
Yeah, agreed, I'm not sure how I didn't notice this.